heat/ml/regression/lasso/lasso.py

```diff
@@ -347,124 +354,124 @@ class PytorchLasso():
+        self.max_iter = max_iter
+        self.tol = tol
+        self.__theta = None
+        self.n_iter = None
 
     @property
-    def coef_(self): 
+    def coef_(self):
         if self.__theta is None:
-            return None 
+            return None
         else:
-            return self.__theta[1:]    
+            return self.__theta[1:]
 
     @property
-    def intercept_(self): 
+    def intercept_(self):
         if self.__theta is None:
             return None
-        else: 
-            return self.__theta[0]      
+        else:
+            return self.__theta[0]
 
     @property
-    def lam(self): 
-         return self.__lam
-       
-    @lam.setter 
+    def lam(self):
+        return self.__lam
+
+    @lam.setter
     def lam(self, arg):
-         self.__lam = arg
+        self.__lam = arg
 
     @property
-    def theta(self): 
-         return self.__theta
+    def theta(self):
+        return self.__theta
 
     def soft_threshold(self, rho):
         """
         Soft threshold operator
- 
+
         Parameters
         ----------
         rho : HeAT tensor, shape (1,)
-            Input model data  
-        out : HeAT tensor, shape (1,) 
-            Thresholded model data 
-        """ 
-        if rho < - self.__lam:
-            return (rho + self.__lam)
-        elif rho >  self.__lam:
-            return (rho - self.__lam)
-        else: 
-            return 0.
+            Input model data
+        out : HeAT tensor, shape (1,)
+            Thresholded model data
+        """
+        if rho < -self.__lam:
+            return rho + self.__lam
+        elif rho > self.__lam:
+            return rho - self.__lam
+        else:
+            return 0.0
 
     def rmse(self, gt, yest):
         """
         Root mean square error (RMSE)
- 
+
         Parameters
         ----------
         gt : HeAT tensor, shape (1,)
-            Input model data  
-        yest : HeAT tensor, shape (1,) 
-            Thresholded model data 
-        """ 
-        return torch.sqrt((torch.mean((gt-yest)**2))).item()
+            Input model data
+        yest : HeAT tensor, shape (1,)
+            Thresholded model data
+        """
+        return torch.sqrt((torch.mean((gt - yest) ** 2))).item()
 
     def fit(self, X, y):
         """
         Fit lasso model with coordinate descent
- 
+
         Parameters
         ----------
         X : HeAT tensor, shape (n_samples, n_features)
-            Input data. 
-        y : HeAT tensor, shape (n_samples,) 
+            Input data.
+        y : HeAT tensor, shape (n_samples,)
             Labels
-        """ 
+        """
         # Get number of model parameters
         _, n = X.shape
 
-        # Initialize model parameters 
+        # Initialize model parameters
         theta = torch.zeros(n, 1)
-        wwtime = np.zeros((self.max_iter, n))
-        # Looping until max number of iterations or convergence 
+        _ = np.zeros((self.max_iter, n))
```

Comment:
so i guess this can be completely removed

Reply:
I would guess so. I kept the statement in order to still have the same side effects, if there are any.